src/kubelet/client/connector.go

```diff
@@ -78,7 +101,9 @@ func (dp defaultConnector) setupLocalConnection(tripperWithBearerToken http.Roun
 			return conn, nil
 		}
 	default:
-		// we were not able to infer the schema and the user did not provided it.
+		dp.logger.Errorf("Checking both HTTP and HTTPS since the schema was not detected automatically, " +
```

Comment:
Is this an `Error`, or perhaps `Warn` or even `Info`?

Reply:
mmm info!